### DISCUSSION TITLE
C/C++

### DISCUSSION THREAD

**Discussion by lemmy** [informalsystems/modelator#174] [General]
What target languages is modelator v2 going to support?  Will the focus remain on rust, or do you expand to, e.g., C/C++, Go, ...?

**Comment by andrey-kuprianov** [comment]
v2 (which will be actually v1.0.0 -- we want to release the first stable version in Q2), will be essentially a rewrite of the current modelator in the new architecture, so it's going to support the same languages as the current one: Rust and Go. But adding support for further languages should be a really easy task in the new design, so C/C++/whatever will follow according to demand. In the new architecture it should be also easy for external contributors to do it themselves; this is one of the main goals of our refactoring.
